lib/common/nodes.c

```diff
@@ -9,8 +9,23 @@
 
 #include <crm_internal.h>
 
+#include <libxml/tree.h>        // xmlNode
 #include <crm/common/nvpair.h>
 
+/*!
+ * \internal
+ * \brief Check whether a node is online
+ *
+ * \param[in] node  Node to check
+ *
+ * \return true if \p node is online, otherwise \false
+ */
+bool
+pcmk_node_is_online(pcmk_node_t *node)
+{
+    return ((node != NULL) && node->details->online)? true : false;
```

Comment:
don't need a conditional operator, just `return (node != NULL) && node->details->online;`

Reply:
thx -- originally I was trying to draw attention to the conversion from gboolean to bool, but that was before `node != NULL` got in there, so now it's not really relevant